### DISCUSSION TITLE
More information about Normalized Systems

### DISCUSSION THREAD

**Discussion by JRBliekendaal** [JRBliekendaal/master-thesis#59] [Q&A] [answered]
@edzob I cannot wait until the lectures on Normalized Systems. I am in need for some background information so I can place Normalized Systems in the right context.. Think about the general idea but also the used principles.

Do you have some information. I cannot find it online.. its a super hidden theory (i get that feeling)

**Comment by edzob** [comment]
https://drive.google.com/drive/folders/1kOW37Hu22apo8IroQpzq4tyB5AmW09v9?usp=sharing

**Comment by edzob** [comment, marked as answer]
you can ask ams for early books, and perhaps the data science student has the books in pdf.
The google link has all the course content.
you can request access
I can see which pdf will give the best synopsis.. tonight or tomorrow

**Reply by edzob** [reply]
I assume you found the slides of JV (in course slides: "AMS MEITA - NS door JV 2019-part 1.pptx" and "AMS MEITA - NS door JV 2019-part 2.pptx"
Slide 26 of part 1 defines the core principles behind NST:
Soc = Seperation of Concerns
DVT = Data Version Transparency
AVT = Action Version Transparency
SoS = Separation of States

**Reply by edzob** [reply]
and I think that "Eessaar 2014 - On Applying Normalized Systems Theory to the Business Architectures of Information Systems.pdf" in NST Papers provides a good introduction in the terminologie.

**Reply by edzob** [reply]
I need to double check if the papers of op t land in the folder of NST papers are relevant.
My hunch itis, since a merger is similar to out/in sourcin :)

**Reply by JRBliekendaal** [reply]
@edzob I thought that op t land used Enterprise Ontology as his lens for mergers. Go to the essential model of both parties and determine what has to be done in the merger. I had a small talk about this with martin two weeks ago. I did not get the impression that he used NST for this.
